lxd/instance/drivers/driver_lxc.go

```diff
@@ -1713,10 +1721,18 @@ func (d *lxc) deviceHandleMounts(mounts []deviceConfig.MountEntryItem) error {
 				}
 			}
 
+			_, err = files.Lstat(mount.TargetPath)
+			if err != nil {
+				err = d.deviceVolatileSetFunc(mount.DevName)(map[string]string{"last_state.created": mount.TargetPath})
+				if err != nil {
+					return fmt.Errorf("Error updating volatile for the device: %w", err)
+				}
+			}
+
 			// Mount it into the container.
-			err := d.insertMount(mount.DevPath, mount.TargetPath, mount.FSType, flags, idmapType)
+			err = d.insertMount(mount.DevPath, mount.TargetPath, mount.FSType, flags, idmapType)
 			if err != nil {
-				return fmt.Errorf("Failed to add mount for device inside container: %s", err)
+				return fmt.Errorf("Failed to add mount for device inside container: %w", err)
```

Comment:
This change should be a separate commit its not related to the commit message

Reply:
fixed